### PR TITLE
Added position and velocity PID

### PR DESCRIPTION
**Pull request by ahcorde**
This PR adds velocity and position PID

This PR is based on this other PR https://github.com/ros-controls/control_toolbox/pull/97

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: gazebo_ros2_control/package.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
 
   <depend>angles</depend>
+  <depend>control_toolbox</depend>
```
Comment (chapulina) [inline gazebo_ros2_control/package.xml:21]: CI failed to find this package

**Comment by ahcorde** [comment]
need a rebase of this PR https://github.com/ros-simulation/gazebo_ros2_control/pull/28 when is merge on master

**Comment by chapulina** [comment]
`joint_trajectory_controller` is failing to build on CI, with a different error from #30

```
 --- stderr: joint_trajectory_controller
/home/ros2_ws/src/ros2_controllers/joint_trajectory_controller/src/trajectory.cpp:22:10: fatal error: rcppmath/clamp.hpp: No such file or directory
   22 | #include "rcppmath/clamp.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by ahcorde** [comment]
> `joint_trajectory_controller` is failing to build on CI, with a different error from #30
> 
> ```
>  --- stderr: joint_trajectory_controller
> /home/ros2_ws/src/ros2_controllers/joint_trajectory_controller/src/trajectory.cpp:22:10: fatal error: rcppmath/clamp.hpp: No such file or directory
>    22 | #include "rcppmath/clamp.hpp"
>       |          ^~~~~~~~~~~~~~~~~~~~
> compilation terminated.
> ```

This is fixed with the new release of Foxy

**Comment by ahcorde** [comment]
friendly ping @chapulina

**Pull request merged**